docs/standard/choosing-core-framework-server.md

```diff
@@ -1,102 +1,77 @@
 ---
 title: Choose between .NET and .NET Framework for server apps
 description: A guide to help you decide which implementation of .NET to use when building a server app.
-ms.date: 08/01/2022
-ms.custom: devdivchpfy22
+ms.date: 06/04/2024
+ms.custom: updateeachrelease
 ---
 # .NET vs. .NET Framework for server apps
 
-There are two supported [.NET implementations](glossary.md#implementation-of-net) for building server-side apps.
+There are two supported [.NET implementations](glossary.md#implementation-of-net) for building server-side apps: .NET and .NET Framework. **The latest .NET version (currently .NET 8) is the preferred version of .NET to use for server development.** The reasons to continue using .NET Framework are specific and limited.
 
-| Implementation | Included versions |
-| - | - |
-| .NET | .NET Core 1.0 - 3.1 </br> .NET 5 and later versions |
-| .NET Framework | .NET Framework 1.0 - 4.8 |
```

Comment:
I think this table does a good job of clarifying what we mean by .NET, which otherwise is in danger of being confusing for those relatively new to .NET.